dev/com.ibm.websphere.appserver.features/visibility/public/jpa-3.0/resources/l10n/com.ibm.websphere.appserver.jpa-3.0.properties

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables support for applications that use application-managed \
+and container-managed JPA. This feature includes Java Persistence API 3.0 specification interfaces \
```

Comment:
replace "Java Persistence" with "Jakarta Persistence" in this file and all other translations